Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -175,6 +183,7 @@ private void SubtractContents(MicrowaveComponent component, FoodRecipePrototype
         private void OnInit(EntityUid uid, MicrowaveComponent component, ComponentInit ags)
         {
             component.Storage = _container.EnsureContainer<Container>(uid, "microwave_entity_container");
+            _deviceLink.EnsureSinkPorts(uid, component.OnPort);
```

Comment:
I think these should go on mapinit not componentinit.

Reply:
will change soon

Reply:
the storage actually had to be in init, it breaks game and tests